dex/elrond_dex_pair/src/fee.rs

```diff
@@ -108,6 +108,11 @@ pub trait FeeModule:
         let mut reserve = self.pair_reserve(token).get();
         reserve += amount;
         self.pair_reserve(token).set(&reserve);
+
+        let mut virtual_reserve = self.pair_virtual_reserve(token, token).get();
```

Comment:
Can use the `update` method instead of get -> modify -> set. Something like this:

```
self.pair_virtual_reserve(token, token).update(|reserve|  *reserve += amount);
```

This makes it so they storage key is built only once instead of twice.

Reply:
Didn't know about update(). Thanks!
Done